src/test/pmem_include/pmem_include.c

```diff
@@ -40,7 +40,7 @@
 
 
 int
-main()
+main(int argc, char *argv[])
```

Comment:
If compiler would start warning about it by itself we would have much bigger problems. There are more instances of unused arguments in our libraries.